CONTRIBUTING.md

```diff
@@ -16,7 +16,8 @@ help also.
 
 The restic project uses the GitHub infrastructure (see the
 [project page](https://github.com/restic/restic)) for all related discussions
-as well as the `#restic` channel on `irc.freenode.net`.
+as well as the [forum](https://forum.restic.net/) and the `#restic` channel
+on `irc.freenode.net`.
```

Comment:
Can we make the `irc.freenode.net` text a link to `https://kiwiirc.com/nextclient/irc.freenode.net/restic`?

Reply:
is changed.